### PR TITLE
docs: add README.md

### PR DESCRIPTION
**Pull request by tzssangglass**
1. add README.md
2. fix startup.sh bug
3. chore

### CODE REVIEW

**Comment by tzssangglass** [comment]
@spacewander @dmsolr pls review

**Comment by tzssangglass** [comment]
development.md & how-it-works.md & the-internal-of-apisix-java-plugin-runner.md will be submitted separately with a new PR

File: README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+with APISIX to run plugins written in java.
+
+The Internal of apisix-java-plugin-runner
+---------------------------------
+
+If you're interested in the internal of apisix-java-plugin-runner, we recommand you
+to read the [the-internal-of-apisix-java-plugin-runner](./docs/the-internal-of-apisix-java-plugin-runner.md),
+it explains the details of communication and protocol conversion with APISIX.
+
+Get Involved to Development
```
Comment (spacewander) [inline README.md:40]: "Involved in"

File: README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+How it Works
+-------------
+
+See [How it Works](./docs/how-it-works.md) to learn how apisix-java-plugin-runner collaborate
+with APISIX to run plugins written in java.
+
+The Internal of apisix-java-plugin-runner
+---------------------------------
+
+If you're interested in the internal of apisix-java-plugin-runner, we recommand you
```
Comment (spacewander) [inline README.md:36]: recommend

File: README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+This project is currently considered as experimental.
+
+Why apisix-java-plugin-runner
+---------------------
+
+APISIX offers many full-featured plugins covering areas such as authentication,
+security, traffic control, serverless, analytics & monitoring, transformations, logging.
+
+It also provides highly extensible api, allowing common phases to be mounted,
```
Comment (spacewander) [inline README.md:20]: API

File: README.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+Runs [Apache APISIX](http://apisix.apache.org/) plugins written in Java.
+Implemented as a Sidecar that accompanies APISIX.
+
+![apisix-java-plugin-runner-overview](./docs/images/apisix-java-plugin-runner-overview.png)
+
+Status
+------
+
+This project is currently considered as experimental.
```
Comment (spacewander) [inline README.md:12]: considered experimental

Comment (tzssangglass) [reply README.md]: update

**Review by spacewander — APPROVED** [review]

**Pull request merged**